### PR TITLE
Winit version 0.30.6

### PR DESCRIPTION
**Pull request by kchibisov**
--

@madsmtm I've excluded a bunch of recent PRs for macOS, except the capslock one, since I'm not entirely sure whether they are relevant for a patch release.

If I've missed something let me know.

### CODE REVIEW

**Review by madsmtm — APPROVED** [review]
I'm happy as long as [the stuff in the milestone](https://github.com/rust-windowing/winit/milestone/28?closed=1) is here (which it seems to be), so 👍 from me.

I actually thought I set https://github.com/rust-windowing/winit/pull/4035 to that as well, but I seem to have set it to v0.31.0, not sure now though, I'll let you decide if you wanna try integrate it.

File: src/changelog/v0.30.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+## 0.30.6
+
+### Added
+
+- On macOS, add `WindowExtMacOS::set_borderless_game` and `WindowAttributesExtMacOS::with_borderless_game`
+  to fully disable the menu bar and dock in Borderless Fullscreen as commonly done in games.
+- On X11, the `window` example now understands the `X11_VISUAL_ID` and `X11_SCREEN_ID` env
+  variables to test the respective modifiers of window creation.
+- On Android, the soft keyboard can now be shown using `Window::set_ime_allowed`.
```
Comment (MarijnS95) [inline src/changelog/v0.30.md:9]: Added https://github.com/rust-windowing/winit/pull/3993 to the milestone.

Comment (kchibisov) [reply src/changelog/v0.30.md:9]: I've added it as a non breaking change, so not sure what it means?

Comment (MarijnS95) [reply src/changelog/v0.30.md:9]: I think it's nice to be able to retroactively see what PRs went into which release?

Comment (kchibisov) [reply src/changelog/v0.30.md:9]: I guess...

**Review by daxpedda — APPROVED** [review]
The Web commit isn't needed, its just some code improvements and has no benefit to the user. But nothing wrong with including it!

**Comment by kchibisov** [comment]
Redox msrv failure is only in example.

**Pull request merged**